### PR TITLE
LPS-133978 Create Tests for UserGroup TableReferenceDefinitions

### PR DESCRIPTION
**Pull request by ctang3**
https://issues.liferay.com/browse/LPS-133978

### CODE REVIEW

File: modules/apps/site/site-memberships-test/src/testIntegration/java/com/liferay/site/memberships/change/tracking/test/UserGroupGroupRoleTableReferenceDefinitionTest.java [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+	@Before
+	@Override
+	public void setUp() throws Exception {
+		super.setUp();
+
+		_role = RoleTestUtil.addRole(RoleConstants.TYPE_SITE);
+
+		_userGroup = UserGroupTestUtil.addUserGroup(group.getGroupId());
+
+		_userLocalService.addUserGroupUser(
```
Comment (Preston-Crary) [inline modules/apps/site/site-memberships-test/src/testIntegration/java/com/liferay/site/memberships/change/tracking/test/UserGroupGroupRoleTableReferenceDefinitionTest.java:65]: Need to delete these after the test. BaseTableReferenceDefinitionTestCase deletes anything under group or added by addCTModel() but Roles and UserGroups do not get cleaned up. You'll need to delete the user group user in order to remove the UserGroup.

File: modules/apps/site/site-memberships-test/src/testIntegration/java/com/liferay/site/memberships/change/tracking/test/UserGroupRoleTableReferenceDefinitionTest.java [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+	@Rule
+	public static final AggregateTestRule aggregateTestRule =
+		new AggregateTestRule(
+			new LiferayIntegrationTestRule(),
+			PermissionCheckerMethodTestRule.INSTANCE);
+
+	@Before
+	@Override
+	public void setUp() throws Exception {
+		super.setUp();
```
Comment (Preston-Crary) [inline modules/apps/site/site-memberships-test/src/testIntegration/java/com/liferay/site/memberships/change/tracking/test/UserGroupRoleTableReferenceDefinitionTest.java:52]: I think setUp() gets called without overriding, can you try removing this method?

**Comment by ctang3** [comment]
Updated, thanks!

**Comment by Preston-Crary** [comment]
ci:forward

**Pull request closed without merging**